src/Stryker.Core/Stryker.Core/StrykerRunner.cs

```diff
@@ -122,7 +122,14 @@ public StrykerRunResult RunMutationTest(IStrykerInputs inputs, ILoggerFactory lo
                 // Restore assemblies
                 foreach (var project in _mutationTestProcesses)
                 {
-                    project.Restore();
+                    try
+                    {
+                        project.Restore();
```

Comment:
I would prefer if this was solved at the source: https://github.com/stryker-mutator/stryker-net/blob/master/src/Stryker.Core/Stryker.Core/ProjectComponents/TestProjects/TestProjectsInfo.cs#L38

No reason for this exception to bubble up all the way to the stryker runner, it should be handled correctly in TestProjectsInfo.